code/modules/holiday/easter.dm

```diff
@@ -60,6 +60,26 @@
 	maxbodytemp = 1500
 	unsuitable_atmos_damage = 0
 
+/mob/living/simple_animal/chicken/rabbit/attack_hand(mob/living/carbon/human/M)
+	. = ..()
+	switch(M.a_intent)
+		if("help")
+			wuv(1,M)
+		if("harm")
+			wuv(-1,M)
+
+/mob/living/simple_animal/chicken/rabbit/proc/wuv(change, mob/M)
+	if(change)
+		if(change > 0)
+			if(M && stat != DEAD)
+				new /obj/effect/temp_visual/heart(loc)
+				emote("me", 1, "hops around happily!")
```

Comment:
Way too much repeated code here. You should really have this function defined on the /mob level, then add a bool var that controls whether or not this interaction happens or not, and set it to TRUE for these mobs. Add two string vars for the reaction they have (positive or negative) and you're all set.

Reply:
I basically had this finished then ran into an enormous bug relating to mob emotes.
I'll try to keep working on this.

Reply:
just keep working as if "me" works, I'll handle the rest